### PR TITLE
add caption to the streetcode catalog page

### PR DESCRIPTION
**Pull request by BohdanBybliv**
**Added "Стріткоди" caption on the streetcode catalog page in right upper corner.**
image
on mobile:
image

### CODE REVIEW

**Review by Maxym1507 — APPROVED** [review]

File: src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss [hunk 1]
```diff
@@ -57,7 +74,17 @@ $loadImg: '@assets/images/catalog/loading.webp';
...
+
+            .streetcodeHeadingContainer {
+                display: flex;
+                justify-content: space-between;
+                margin-top: pxToRem(20px);
+                .headingFlexItem {
+                    display: inline-block;
+                }
+            }
+
```
Comment (mykola-krai) [inline src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss:87]: Duplicate for:
lines 30-38.

File: src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss [hunk 2]
```diff
@@ -57,7 +74,17 @@ $loadImg: '@assets/images/catalog/loading.webp';
         .streetcodeCatalogWrapper {
             padding: pxToRem(80px) pxToRem(13px) 0;
             margin-top: 0;
-    
+            width: 100%;
```
Comment (mykola-krai) [inline src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss:77]: Duplicate for:
line 28 width: 100%;

File: src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss [hunk 3]
```diff
@@ -19,6 +26,16 @@ $loadImg: '@assets/images/catalog/loading.webp';
         margin-top: 5.125rem;
         padding: pxToRem(10px) 10% pxToRem(100px) 10%;
         width: 100%;
+
+        .streetcodeHeadingContainer {
+            display: flex;
+            justify-content: space-between;
+            margin-top: pxToRem(20px);
```
Comment (mykola-krai) [inline src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss:33]: Check margins for mobile and tablet version in figma.

**Review by mykola-krai — CHANGES_REQUESTED** [review]
If an element styles are stated outside the media rules, they work for each media rule.
So they should not be duplicated, unless necessarily or they must be changed.

File: src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss (hunk 3 above)
Comment (BohdanBybliv) [reply src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss:39]: I fixed it

File: src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss [hunk 4]
```diff
@@ -7,6 +7,21 @@
 $bgImg: '@assets/images/catalog/background.webp';
 $loadImg: '@assets/images/catalog/loading.webp';
 
+.streetcodeCatalogCaption {
+    @include mut.with-font($font-family: f.$roboto-font, $font-weight: 300, $font-size: 12px);
+    line-height: 14px;
+    text-transform: uppercase;
+    color: #891F16;
```
Comment (KateYatsiuk) [inline src/features/StreetcodeCatalogPage/StreetcodeCatalog.styles.scss:14]: Consider using variables for colors, we have an appropriate file for them

**Review by KateYatsiuk — APPROVED** [review]
Please, pay attention to my comment

**Review by mykola-krai — APPROVED** [review]

**Pull request merged**